packages/components/src/utils/rtl.js

```diff
@@ -79,3 +79,6 @@ export function rtl( ltrStyles = {}, rtlStyles ) {
 		return isRTL() ? css( convertLTRToRTL( ltrStyles ) ) : css( ltrStyles );
 	};
 }
+
+// Useful when in need of observing RTL/LTR changes in the current locale.
+rtl.watch = () => isRTL();
```

Comment:
We should probably have a more descriptive comment about the context in which this is useful:

```suggestion
/**
 * Call this in the `useMemo` dependency array to ensure that subsequent renders will
 * cause rtl styles to update based on the `isRTL` return value even if all other dependencies
 * remain the same.
 *
 * @example
 * const styles = useMemo( () => {
 *   return css`
 *     ${ rtl( { marginRight: '10px' } ) }
 *   `;
 * }, [ rtl.watch() ] );
 */
rtl.watch = () => isRTL();
```

Reply:
Very good point, thank you for suggesting a great comment